translations/translator.py

```diff
@@ -191,7 +320,7 @@ def get_added_items_in_bundle(self):
         default = bundle_as_dictionary[default_locale_file]
 
         if default != snapshot:
-            new_values = [f'{key}: {val}' for key, val in default.items() if default[key] not in snapshot.values()]
+            new_values = {key: val for key, val in default.items() if default[key] not in snapshot.values()}
             '''
                 TODO: Ignoring the capability to actually make an inplace
```

Comment:
Can't we just add back the `updated_items` logic we had previously?

Reply:
We went through some discussion to see how we could minimize the amount of messages that we track, and realized that updated and new are the same from the perspective of sending and receiving translations. There are more details around the logic adjustment, we can talk more to explain it.